nebula/ui/components/VPNConnectionInfoScreen.qml

```diff
@@ -51,9 +51,9 @@ Rectangle {
         },
         State {
             name: "open-loading"
-            when: (VPNConnectionBenchmark.state !== VPNConnectionBenchmark.StateInitial
-                && VPNConnectionBenchmark.state !== VPNConnectionBenchmark.StateReady
-                && VPNConnectionBenchmark.state !== VPNConnectionBenchmark.StateError)
+            when: (MZModules["vpn"].connectionBenchmark.state !== MZModules["vpn"].connectionBenchmark.StateInitial
```

Comment:
If this starts to feel verbose we could do something like `property var connectionBenchmark = MZModules["vpn"].connectionBenchmark` at the top then `connectionBenchmark.state` throughout.